addons/web/static/src/views/kanban/kanban_record.js

```diff
@@ -166,30 +168,37 @@ export class KanbanRecord extends Component {
     static defaultProps = {
         colors: COLORS,
         deleteRecord: () => {},
+        getSelection: () => [],
         archiveRecord: () => {},
         openRecord: () => {},
+        selectionAvailable: false,
+        toggleSelection: () => {},
     };
     static props = [
         "archInfo",
         "canResequence?",
         "colors?",
         "Compiler?",
         "forceGlobalClick?",
+        "getSelection?",
         "group?",
         "groupByField?",
         "deleteRecord?",
         "archiveRecord?",
         "openRecord?",
         "readonly?",
         "record",
+        "selectionAvailable?",
         "progressBarState?",
+        "toggleSelection?",
```

Comment:
same here (a callback is fine, but should it allow to toggle the selection of other records?) I would bind it to this record only.

Reply:
This is required to enable "shift + click" and toggle the selection of a range of records : `this.props.toggleSelection(this.props.record, ev.shiftKey);`